release/build.sh

```diff
@@ -36,6 +36,7 @@ fi
 
 PRERELEASE_DOCKER_HUB=${PRERELEASE_DOCKER_HUB:-gcr.io/istio-prerelease-testing}
 GCS_BUCKET=${GCS_BUCKET:-istio-prerelease/prerelease}
+SBOM_OUTPUT_URI="https://storage.googleapis.com/${GCS_BUCKET}/releases"
```

Comment:
Do we want this value to be overwrite able as it isn't as written here? If not, then we could include it in the URI below and save a line.